packages/clarity-js/src/layout/encode.ts

```diff
@@ -38,24 +38,23 @@ export default async function (type: Event, timer: Timer = null, ts: number = nu
             region.reset();
             break;
         case Event.StyleSheetAdoption:
-            for (let entry of style.state) {
+        case Event.StyleSheetUpdate:
```

Comment:
is this intentional that we process both when one happens?

Reply:
probably unnecessary - but I thought this was still safer and set us up for the future better